tests/test_browser.py

```diff
@@ -4817,6 +4817,10 @@ def test_emscripten_performance_now(self):
   def test_embind_with_pthreads(self):
     self.btest('embind_with_pthreads.cpp', '1', args=['--bind', '-s', 'USE_PTHREADS=1', '-s', 'PROXY_TO_PTHREAD=1'])
 
+  @no_fastcomp("no asyncify support")
+  def test_embind_with_asyncify(self):
+    self.btest('embind_with_asyncify.cpp', '1', args=['--bind'] + self.get_async_args())
```

Comment:
It would be better to put the test in the main test suite, `core` - that way it will run in multiple optimization modes.

Reply:
This test uses `fetch` API - will that work in `core` tests? I've assumed that `test_browser` is for tests that rely on browser-specific APIs (e.g. `embind_with_pthreads` above).

Reply:
Oh, I didn't notice it used a browser API.

One option would be to use an async API that works off the web. Or, is fetch intentionally used here as a specific example? If we think it has value in being more realistic, maybe it's fine here then.

Reply:
> One option would be to use an async API that works off the web. Or, is fetch intentionally used here as a specific example?

It was mostly to be realistic, but I think I can just use `Promise.resolve(...)` here too if there's value in cross-platform testing.

Reply:
Hmm, might be fine as it is then. There is value to realistic testing too.

Reply:
In principle there's value in having a realistic sample test, but also an unrealistic easier-to-test test.

Can be done in a followup PR though; this gets enough coverage IMO